lineman/app/components/thread_preview_collection/thread_preview_collection.scss

```diff
@@ -1,4 +1,4 @@
-.thread-previews-container {
+.inbox-page__groups .thread-previews-container, .dashboard-page__collections .thread-previews-container {
```

Comment:
Don't need to change this now.. (i'll include it in the tech-manual) but I think with my preferred language "container" would be "panel" a panel is something that can be put inside a card.

Also I reckon we might need to come up with a a rule in the future... something like all cards have no padding by default and you need to add your own with one of these handy mixins.

For now this is super. If you're happy then we can merge